topology/sof/pipe-low-latency-playback.m4

```diff
@@ -115,5 +115,5 @@ indir(`define', concat(`PIPELINE_PCM_', PIPELINE_ID), Low Latency Playback PCM_I
 
 
 # PCM capabilities supported by FW
-PCM_CAPABILITIES(Low Latency Playback PCM_ID, `S32_LE,S24_LE,S16_LE', 48000, 48000, 2, 2, 2, 16, 192, 16384, 65536, 65536)
+PCM_CAPABILITIES(Low Latency Playback PCM_ID, `S32_LE,S24_LE,S16_LE', 8000, 192000, 2, 2, 2, 16, 192, 16384, 65536, 65536)
```

Comment:
Has this been tested ? The range was locked at 48kHz for some pipelines that had no SRC where the SSP port was fixed at 48kHz. Non 48kHz rates require SRC when the SSP is running at a different rate. I think @singalsu has some examples.

Reply:
Topology pipe-src-playback.m4 has this range. There would be also need for this same change for capture from DMIC for other than 48 kHz rate. With DMIC I've successfully recorded at 8/16/32/48/64/96 kHz. 24 kHz should work too but's there some ALSA issue.

Reply:
@singalsu can you craete kernel issues for DMIC 24kHz.

Reply:
I was  little bit unsure what Pierre wanted in his issue. The pipelines work with this patch, but if you play with different sample rate than DAI without plughw, playback will be funny->slower/faster. OTH I can add this only to the media pipeline, which will then have src just for doing nothing (if you play at DAI rate).

Reply:
@juimonen if the playback is wrong speed it should be rejected by the FW. I think @plbossart was meaning it's difficult to represent 44.1kHz with a 1ms scheduling tick as 44100 / 1000 is 44.1 i.e. not an integer. The FW copier needs to copy an integer number of samples. i.e. 44.1kHz would work if it was scheduled every 441ms as 100 samples would be copied per schedule.

Reply:
@juimonen you could alos put a check in that rate is divisible by scheduling period.